robot_assignments/web/jet2.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     String
+
+*** Variables ***
+${urlLogin}    http://125.26.15.143:13132/ 
+${urlhome}      http://125.26.15.143:13132/home.html
+${username1}    helloworld
+${password1}    1234
+${username2}    oz4899
+${password2}    1234
+${loginIDFieldXpath}    //input[@id='username']
+${loginPasswordFieldXpath}      //input[@id='password']
+${loginButtonXpath}     //button[@type='submit']
+${homePageContentXpath}     //div[@class='container']/div[@class='box']
+${searchFieldXpath}     //div[@class='column is-four-fifths']/input 
+${searchButtonXpath}       //button[@id='searchbtn'] 
+${itemListXpath}        //div[@class='card-content p-4']/p
+${noResultFoundXpath}       //div[@class='box']/div/div
+${firstItemXpath}       //div[@class='box']/div/div[1]
+${addToCartButtonXpath}      //button[@class='button is-info is-size-5']
+${itemQuantityXpath}     //span[@id='lblCartCount']/b
+${plusButtonXpath}      //div[@id='remove']/button
+${cartHoverXpath}       //div[@class='column']/div/div[@style='text-align: center;']/div/div[@class='dropdown-trigger']
+${clearCartButtonXpath}     //button[@class='button is-danger']
+
+*** Keywords ***
+Open Website
+    Open Browser  url=${urlLogin}  browser=chrome
+
+Login Website   
+    Input Text   ${loginIDFieldXpath}    ${username2}
+    Input Text   ${loginPasswordFieldXpath}      ${password2}
+    Click Element   ${loginButtonXpath}
+    Wait Until Page Contains Element    ${homePageContentXpath}
+    ${currentUrl}=     Get Location
+    Should be equal      ${urlhome}     ${currentUrl}
+
+*** Test cases ***
+Testcase 01 - Verify that user cannot login with wrong username
+    Open Website
+    Input Text   ${loginIDFieldXpath}     ${username1}
+    Input Text   ${loginPasswordFieldXpath}     ${password1}
+    Click Element   ${loginButtonXpath}
+    ${currentUrl}=     Get Location
+    Should be equal      ${urlLogin}     ${currentUrl}
+    sleep   1s
+    Close Browser
+
+Testcase 02 - Verify that user can login to the system when username are correct
+    Open Website   
+    Login Website
+    sleep   1s
+    Close Browser
+
+Testcase 03 - Verify that search function are working correctly
+    Open Website   
+    Login Website
+
+    Input Text      ${searchFieldXpath}     abcd
+    Click Element   ${searchButtonXpath}  
+    sleep   1s
```

Comment:
Same comment for all sleep 1s

Can you think of the better way to avoid sleep and still verify the result is correct?